client/my-sites/customer-home/welcome-banner.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Card, Button } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+
+export const WelcomeBanner = ( { translate } ) => {
+	return (
+		<div className="customer-home__welcome-banner">
```

Comment:
If we're moving away from the button, there's a dismissable card component we can use, that also persists dismiss state:

https://wpcalypso.wordpress.com/devdocs/blocks/dismissible-card